06-developer-software-guide.Rmd

```diff
@@ -6,6 +6,20 @@ dependencies described in the [user installation
 guide](#user-installation-guide) which you should ensure are installed
 first.
 
+### Codespaces
+
+An alternative to setting up this software locally is using [GitHub Codespaces](https://docs.github.com/en/codespaces).
+
+The FIMS repository already has a file to set up all of the needed software, so there is no need to install anything locally.
```

Comment:
Change to
`An alternative to setting up this software locally is using [GitHub Codespaces](https://docs.github.com/en/codespaces), which does not require you to install anything locally.`

Reply:
And we could potentially delete the instructions below and the screen shot if we just link to [these instructions on GitHub](https://docs.github.com/en/codespaces/developing-in-a-codespace/creating-a-codespace-for-a-repository#creating-a-codespace-for-a-repository).

Reply:
Ah, thanks! I didn't see the "webui" tab on this page. I think this will link directly to the instructions: https://docs.github.com/en/codespaces/developing-in-a-codespace/creating-a-codespace-for-a-repository?tool=webui#creating-a-codespace-for-a-repository